shared/src/main/kotlin/com/egm/stellio/shared/model/ErrorResponse.kt

```diff
@@ -40,10 +40,9 @@ data class TooComplexQueryResponse(override val detail: String) : ErrorResponse(
 
 data class TooManyResultsResponse(override val detail: String) : ErrorResponse(
     ErrorType.TOO_MANY_RESULTS.type,
-    """
-    The query associated to the operation is producing so many results that can exhaust client or server resources. 
-    It should be made more restrictive
-    """,
+    "The query associated to the operation is producing so many results " +
+        "that can exhaust client or server resources. " +
+        "It should be made more restrictive",
```

Comment:
use multi-line string, as it was before. easier to read, no?

Reply:
with the multi-line string the tests fail because the json string can't match. I tried \n but it still failed

Reply:
indeed, you would have to regex replace the `\n` in the value for the final result to be a single-line string (which is what is expected in the API response). not a problem but it adds not necessary overhead, keep it as it is.